src/clj/fluree/db/query/api.cljc

```diff
@@ -34,10 +34,10 @@
     did (assoc :did did :issuer did)))
 
 (defn restrict-db
-  [db t opts]
+  [db t {:keys [did default-allow?]}]
   (go-try
-    (let [db*  (if-let [did (:did opts)]
-                 (<? (perm/wrap-identity-policy db did true nil))
+    (let [db*  (if did
+                 (<? (perm/wrap-identity-policy db did default-allow? nil))
```

Comment:
I think this change may break some `fluree/server` tests. You should verify you don't need a new PR to `fluree/server` to still work based on the changes here. If so, that PR should also get queued up.

Reply:
I ran the `fluree/server` tests pointed at these latest changes and they all passed.